### PR TITLE
Minor UI updates and fixes

### PR DESCRIPTION
**Pull request by mm-vasyl**
adds custom scrollbar, fixes fieldset overflow for filters, fixes html empty button appearing on banner card controls

### CODE REVIEW

**Comment by mm-paulie** [comment]
Can you put some screenshots to explain the bugs? Thanks :)

**Comment by mm-vasyl** [comment]
Hey sure!
Here's before and after re filters fieldset:
image
image

File: src/preview/src/App.css [hunk 1]
```diff
@@ -1,43 +1,18 @@
...
   }
-  to {
-    transform: rotate(360deg);
+
+  &::-webkit-scrollbar-thumb {
+    background-color: #fff;
+    border-radius: 1rem;
+    border: 3px solid #121212;
   }
 }
```
Comment (mm-paulie) [inline src/preview/src/App.css:18]: I see, that .App related stuff is not used anymore right? @mm-vasyl Good notice

File: src/preview/src/components/AdPreview.js [hunk 2]
```diff
@@ -111,7 +111,7 @@ export const AdPreview = (props) => {
           <Box>
             <Box display="flex" flexWrap="wrap">
               {Object.keys(ad.output).map((extension) => {
-                if (ad.output[extension])
+                if (ad.output[extension] && extension != 'html')
                   return (
                     <Tooltip key={extension} title={`Download ${extension.toUpperCase()} ${!ad.output[extension]?.url ? "(loading)" : ""}`}>
                       <span>
```
Comment (mm-paulie) [inline src/preview/src/components/AdPreview.js:117]: Never noticed this created button, nice catch!

File: src/preview/src/components/Previews.js [hunk 3]
```diff
@@ -171,7 +171,7 @@ export default function Previews({ data }) {
               multiple
               value={getSelectedFilters()}
               onChange={handleChangeFilter}
-              input={<OutlinedInput id="select-multiple-chip" label="Chip" />}
+              input={<OutlinedInput id="select-multiple-chip" label="Filters" />}
               renderValue={(selected) => (
                 <Box sx={{ display: "flex", flexWrap: "wrap", gap: 0.5 }}>
                   {selected.map((value) => (
```
Comment (mm-paulie) [inline src/preview/src/components/Previews.js:177]: Another great catch!

**Review by mm-paulie — APPROVED** [review]
Perfect catches, nothing more to add

File: src/preview/src/App.css (hunk 1 above)
Comment (mm-vasyl) [reply src/preview/src/App.css:18]: Yup, it had some default unused react styles

File: src/preview/src/App.css
Comment (mm-paulie) [inline src/preview/src/App.css:1]: What kind of other elements do you want to give a custom scrollbar?

Comment (mm-vasyl) [reply src/preview/src/App.css:1]: I noticed that filters sometimes are too big so they receive an ugly scrollbar

**Review by mm-paulie — APPROVED** [review]

File: src/preview/src/App.css
Comment (mm-paulie) [reply src/preview/src/App.css:1]: image
Looks good!

**Review by Syridiana — APPROVED** [review]
Everything looks good to me !
Thanks Vasyl :)

**Review by Syridiana — APPROVED** [review]

**Comment by PabloCarreira** [comment]
Approved, everything looking nice on my end

**Pull request merged**